### PR TITLE
Create ui-c8y-1019-19-6-move-help-text-to-c-8-y-messages-in-map-widget.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58126](https://cumulocity.atlassian.net/browse/MTM-58126)
Original PR: [6097](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6097)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-19-6-move-help-text-to-c-8-y-messages-in-map-widget.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: move help text to c8y-messages in map widget (#6025) [GRAFT][release/cd] (#6097)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-19-6-move-help-text-to-c-8-y-messages-in-map-widget.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Map widget help text moved to c8y-messages
```

File: content/change-logs/application-enablement/ui-c8y-1019-19-6-move-help-text-to-c-8-y-messages-in-map-widget.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58126
+version: 1019.19.6
+---
+move help text to c8y-messages in map widget (#6025) [GRAFT][release/cd] (#6097)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-19-6-move-help-text-to-c-8-y-messages-in-map-widget.md:17]: The Claude AI suggested to use the following body:
```suggestion
The map widget in the Cockpit application previously displayed its help text directly inside the widget. To provide a more consistent user experience, the help text has now been moved to a c8y-messages component. This change aligns the map widget with other widgets in the application that also use c8y-messages for displaying help information. The functionality of the map widget remains the same, only the location of the help text has been updated.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by eniosultan** [comment]
@BeateRixen  We can close this. This is not relevant for release notes..

**Pull request closed without merging**